tracer/src/Datadog.Trace/AppSec/SecuritySettings.cs

```diff
@@ -27,13 +27,13 @@ public SecuritySettings(IConfigurationSource? source, IConfigurationTelemetry te
         {
             source ??= NullConfigurationSource.Instance;
             var config = new ConfigurationBuilder(source, telemetry);
-            BlockedHtmlTemplate = config
+            BlockedHtmlTemplatePath = config
                                  .WithKeys(ConfigurationKeys.AppSec.HtmlBlockedTemplate)
-                                 .AsRedactedString(SecurityConstants.BlockedHtmlTemplate); // Redacted because it's huge
+                                 .AsRedactedString(); // Redacted because it's huge
 
-            BlockedJsonTemplate = config
+            BlockedJsonTemplatePath = config
                                  .WithKeys(ConfigurationKeys.AppSec.JsonBlockedTemplate)
-                                 .AsString(SecurityConstants.BlockedJsonTemplate);
+                                 .AsString();
```

Comment:
For me it's not clear what's the default value. Should we sanitize the platform dependant path separators?

Reply:
The default value is null isn't it? Doesn't seem like a problem to me from a telemetry PoV

> Should we sanitize the platform dependant path separators

This is a good question. We don't do in other settings either, I wonder if we should. I'm not very concerned about that though, as it's up to customers, and would just count as "bad config" otherwise.

Reply:
I'm okay with not sanitizing and requiring customers to put the correct value.